python/lsst/meas/astrom/pessimistic_pattern_matcher_b_3D.py

```diff
@@ -573,17 +573,18 @@ def _construct_pattern_and_shift_rot_matrix(self, src_pattern_array,
                     src_pattern_array[0], ref_center, src_delta_array[0],
                     ref_delta, cos_shift, max_cos_rot_sq)
                 if test_rot_struct.cos_rot_sq is None or \
+                   test_rot_struct.proj_ref_ctr_delta is None or \
                    test_rot_struct.shift_matrix is None:
                     continue
 
                 # Get the data from the return struct.
                 cos_rot_sq = test_rot_struct.cos_rot_sq
+                proj_ref_ctr_delta = test_rot_struct.proj_ref_ctr_delta
```

Comment:
This name is not very clear compared to its description: "Plane projected 3 vector of the first candidate pair of the pinwheel." Is ctr meant to be center? Consider something starting with vector_proj.

Reply:
I've changed the description and hopefully that improves things.